common/src/main/java/com/microsoft/aad/adal/IBrokerAccountService.java

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Without `.aidl` file , will the broker communication work ? I see the interface is generated here but want to make sure 
Are we seeing warnings due to this regeneration?
Also we do another aidl for MSAL `IMicrosoftAuthService` , isn't that throwing any warnings/errors?